servers/Fabric.Mcp.Server/vscode/package.json

```diff
@@ -0,0 +1,130 @@
+        "Fabric",
+        "Chat",
+        "Extension Packs"
+    ],
+    "keywords": [
+        "fabric",
+        "agent",
+        "chat",
+        "copilot",
+        "language-model-tools",
+        "mcp",
+        "model context protocol",
+        "ai"
+    ],
+    "bugs": {
+        "url": "https://github.com/microsoft/mcp/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/mcp"
+    },
+    "activationEvents": [
+        "onStartupFinished"
+    ],
+    "main": "main.js",
+    "contributes": {
+        "mcpServerDefinitionProviders": [
+            {
+                "id": "fabricMcpProvider",
+                "label": "Fabric MCP Server Provider"
+            }
+        ],
+        "configuration": {
+            "type": "object",
+            "title": "Fabric MCP",
+            "properties": {
+                "fabricMcp.enabledServices": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "publicapis"
+                        ],
+                        "markdownEnumDescriptions": [
+                            "Fabric public APIs — Container registry management."
+                        ]
+                    },
+                    "uniqueItems": true,
+                    "markdownDescription": "Service namespaces to enable. Leave empty to enable all. Choose from the list to avoid typing errors. **Note:** Changes require restarting the MCP server if MCP Autostart is not configured (Command Palette → MCP: List Servers → Fabric MCP → Start/Restart)."
+
+                },
+                "fabricMcp.serverMode": {
+                    "type": "string",
+                    "enum": ["single", "namespace", "all"],
+                    "default": "all",
+                    "markdownDescription": "Server Mode determines how tools are exposed: `single` collapses every tool (100+) into one (1) single tool that routes internally, `all` (default) collapses all tools down into logical Fabric service namespace grouping (about 30 tools), while `all` exposes every MCP tool directly to the MCP client. We recommend all as the right balance between MCP tool count and tool selection accuracy. **Note:** Changes require restarting the MCP server if MCP Autostart is not configured (Command Palette → MCP: List Servers → Fabric MCP → Start/Restart)."
+
+                },
+                "fabricMcp.readOnly": {
+                    "type": "boolean",
+                    "default": false,
+                    "markdownDescription": "Start the Fabric MCP server in read-only mode (operations that modify resources will be disabled). **Note:** Changes require restarting the MCP server if MCP Autostart is not configured (Command Palette → MCP: List Servers → Fabric MCP → Start/Restart)."
+                }
+            }
+        }
+    },
+    "extensionPack": [
```

Comment:
This can be removed, I don't think fabric VSIX needs to be bundled with GHCP4A extension.